plugins/se/client.h

```diff
@@ -0,0 +1,174 @@
+// Copyright 2019 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#pragma once
```

Comment:
This is not standard c++ (see https://stackoverflow.com/questions/23696115/is-pragma-once-part-of-the-c11-standard).
I'd replace this with the traditional `#ifndef` header-guard we use in the rest of Mumble's code-base as well